homeassistant/components/dynalite/__init__.py

```diff
@@ -62,7 +88,10 @@ def num_string(value):
         vol.Optional(CONF_POLLTIMER, default=1.0): vol.Coerce(float),
         vol.Optional(CONF_AREA): AREA_SCHEMA,
         vol.Optional(CONF_DEFAULT): PLATFORM_DEFAULTS_SCHEMA,
-        vol.Optional(CONF_ACTIVE, default=False): vol.Coerce(bool),
+        vol.Optional(CONF_ACTIVE, default=False): vol.Any(
+            CONF_ACTIVE_ON, CONF_ACTIVE_OFF, CONF_ACTIVE_INIT, bool
```

Comment:
`bool` should probably be `cv.boolean`

Reply:
Also, isn't it confusing that you allow string values and boolean values here ?

Reply:
I wanted to keep it backwards compatible since there is a third option. I can move it to only the CONF_* as true and false get routed to on and off

Reply:
ok, makes sense.